### DISCUSSION TITLE
Custom Object Detection works with RetinaNet but fails with Fast R-CNN or Mask R-CNN

### DISCUSSION THREAD

**Discussion by gul15** [facebookresearch/detectron2#3923] [Q&A]
Hello everyone, I have been trying to develop a vertebra detector using detectron2. I developed a simple one using RetinaNet from [detectron2 model zoo](https://github.com/facebookresearch/detectron2/blob/main/MODEL_ZOO.md). 
The input annotated data looks like this:
![image](https://user-images.githubusercontent.com/67880686/151120442-71efef1b-6a8e-4b2a-b0f2-7e2e0de549ca.png)

Also the output from RetinaNet:
![image](https://user-images.githubusercontent.com/67880686/151120752-381b5adc-5363-4e6d-bb5a-24fde8c6ee1f.png)

But when I try to use other detection models such as Faster R-CNN R50-FPN or Instance Segmentation models it does not give sensible outputs.

I would appriciate any help! Thanks.

**Comment by zhanghang1989** [comment]
This might be related to the anchors set up. You may try to adjust the anchor settings
